### PR TITLE
check if cow order is fulfilled to calculate avg price

### PR DESCRIPTION
**Pull request by Diogomartf**
Fixes:
image

### CODE REVIEW

File: packages/app/models/stack-order/stack-order.ts [hunk 1]
```diff
@@ -16,6 +17,8 @@ export const calculateStackAveragePrice = (order: StackOrder) => {
 
   order.cowOrders.forEach((cowOrder) => {
     if (cowOrder.executedBuyAmount === "0") return;
+    // @ts-expect-error
+    if (cowOrder.status !== CowOrderStatus.FULFILLED) return;
```
Comment (ElRodrigote) [inline packages/app/models/stack-order/stack-order.ts:21]: Our `CowOrderStatus` and CoW SDK `OrderStatus` have the same values, but TS tries to infer the types from two different enums and has collisions. IN THIS CASE, we can drop our `CowOrderStatus` and simply use the one from CoW SDK to fix this issue

```suggestion
    if (cowOrder.status !== OrderStatus.FULFILLED) return;
```

**Review by ElRodrigote — APPROVED** [review]

File: packages/app/models/stack-order/stack-order.ts (hunk 1 above)
Comment (Diogomartf) [reply packages/app/models/stack-order/stack-order.ts:21]: Yeah, it should be that, that was my first thought. But using `OrderStatus` breaks compilation unexpectedly.
 
image

Comment (Diogomartf) [reply packages/app/models/stack-order/stack-order.ts]: Was able to use `OrderStatus` 🥳

**Review by berteotti — APPROVED** [review]
Awesome, much cleaner!

**Comment by berteotti** [comment]
Can you rebase the branch ? Looks like build is failling too, probably rebase can solve this

**Pull request merged**